libevmasm/Assembly.cpp

```diff
@@ -205,7 +205,8 @@ ostream& Assembly::streamAsm(ostream& _out, string const& _prefix, StringMap con
 	{
 		_out << _prefix << "stop" << endl;
 		for (auto const& i: m_data)
-			assertThrow(u256(i.first) < m_subs.size(), AssemblyException, "Data not yet implemented.");
```

Comment:
I'd like to create an issue about this.  I think, what's missing is a way to distinguish data and subassembly in this array.